src/Form/MessageSettingsForm.php

```diff
@@ -132,15 +128,15 @@ public function buildForm(array $form, FormStateInterface $form_state) {
    * {@inheritdoc}
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
-    $config = $this->config('message.settings');
+    parent::submitForm($form, $form_state);
+
+    $config = $this->configFactory->getEditable('message.settings');
```

Comment:
@damiankloip you are so knowledgable, I'm starting to have feeling towards you! 💘